build.sh

```diff
@@ -19,8 +19,26 @@ esac
 shift # past argument or value
 done
 
+# Flatten resulting image.
+function flatten() {
+  local image="$1"
+  echo "Flattening image '$image'..."
+
+  # We have to rebuild some values since we're flattening with
```

Comment:
Can you take another stab at this comment? It doesn't really help me understand what's going on here.